k9mail/src/main/java/com/fsck/k9/message/IdentityHeaderBuilder.java

```diff
@@ -10,6 +10,7 @@
 import com.fsck.k9.K9;
 import com.fsck.k9.activity.MessageReference;
 import com.fsck.k9.mail.internet.TextBody;
+import com.fsck.k9.message.quote.InsertableHtmlContent;
```

Comment:
The class was moved out of this package which is why now an import statement is necessary. Same for the other occurrences.